src/sentry/models/group.py

```diff
@@ -353,6 +353,18 @@ def get_issues_mapping(
         }
 
 
+# need to rebase and remove this
+class GroupType(Enum):
+    ERROR = 1
+    PERFORMANCE_N_PLUS_ONE = 1000
+    PERFORMANCE_SLOW_SPAN = 1001
+
+
+class IssueCategory(Enum):
```

Comment:
Should we keep the naming consistent here? 
`GroupType` and `GroupCategory` or
`IssueType` and `IssueCategory`

Reply:
Ah yeah the naming changed since I made the migration on Monday, I can update it since it's blocked.